templates/hp_procurve_show_mac-address.template

```diff
@@ -0,0 +1,9 @@
+Value MAC ([0-9a-fA-F]{6}-[0-9a-fA-F]{6})
+Value PORT (\S+)
+Value VLAN (\d+)
+
+Start
+  ^.*MAC Address -> MAC
+
+MAC
+  ^\s+${MAC}\s+${PORT}\s+${VLAN} -> Record
```

Comment:
Do you mind adding the following to the end of states for all templates:
```
  ^\s*$$
  ^. -> Error
```

It is possible implementing the above will cause errors as all lines need to be accounted for. If you have any questions regarding this, let me know and I'll be more than happy to work with you on explaining/fixing this.

Do you mind removing any templates not related to the command this PR is for? Also, please make sure all files/references to commands include all underscores.

These changes will need to be done on all PRs, but once tests pass and I take one last look, I'll get them merged in.

Reply:
Removed not applicable files and added the `-> Error` as requested.

Reply:
Everything should be fixed now.